docusaurus/docs-internals/editor-controls/data-controls/textarea.md

```diff
@@ -0,0 +1,43 @@
+| :------------: | :-----: | :-----: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|      `id`      | string  |    -    | The identifier of the key where the switch will save your data                                                                                                                                                                                                                   |
+|     `type`     | string  |    -    | Type should be `"switch"` to use this control                                                                                                                                                                                                                                    |
+|    `label`     | string  |    -    | The label displayed on the left side of the control                                                                                                                                                                                                                              |
+|   `position`   | number  |    -    | The position of the control in toolbar                                                                                                                                                                                                                                           |
+|   `devices`    | string  | `"all"` | Define the devices where the control will be rendered. Options: `"all"`, `"desktop"`, `"responsive"`. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+|   `disabled`   | boolean |  false  | Configure the condition under which the control is disabled or enabled.                                                                                                                                                                                                          |
+| `placeholder`  | string  |    -    | The placeholder text that appears when the field is empty.                                                                                                                                                                                                                       |
+| `config.lines` | number  |    -    | Number of lines.                                                                                                                                                                                                                                                                 |
+
```

Comment:
1. `textarea` also has the `config.size` property with `"auto"` default

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/d3687ae6-323c-4498-a4a1-a9f722130143)

2. check also if the `textarea` can render the helper, if it can then add also to this table